clients/java/src/main/java/io/camunda/zeebe/client/api/command/CreateProcessInstanceCommandStep1.java

```diff
@@ -101,6 +101,17 @@ interface CreateProcessInstanceCommandStep3 extends FinalCommandStep<ProcessInst
      */
     CreateProcessInstanceCommandStep3 variables(Object variables);
 
+    /**
+     * Overrides the default start position of the process. Calling this method will make the
+     * process start at the given {@code elementId}, if possible. This method can be called more
+     * than once to simultaneously start ar different elements in different branches of the process
+     *
+     * @param elementId elementId
```

Comment:
:wrench: We could add more description for the parameter.

```suggestion
     * @param elementId the id of the BPMN element where to start the process instance
```